src/robot/conf/languages.py

```diff
@@ -20,27 +20,31 @@
 
 
 class Languages:
+    """Keeps a list of languages and unifies the translations in the properties.
 
-    def __init__(self, languages=None):
+    Parameters:
+        languages: a language or a list of languages. Can be the name, the code or an instance.
+        add_default: if True the default language (En) and some aliases is (Default: True)
```

Comment:
:+1: for doc strings. Should preferably use the [Sphinx style](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html) for specifying parameters.